product_docs/docs/pge/17/release_notes/rel_notes17.3.mdx

```diff
@@ -0,0 +1,16 @@
+---
+title: EDB Postgres Extended Server 17.3 release notes
+navTitle: "Version 17.3"
+---
+
+Released: 13 Feb 2025
+
+EDB Postgres Extended Server 17.3 includes the following enhancements and bug fixes:
+
+!!! Note 
+The `adminpack` packages are end of life in EDB Postgres Extended Server 17 and later.
```

Comment:
For EPAS, we have a note regarding the deprecation of DRITA and pgAgent, but AFAIK those aren't available for PGE. 
Hence I have only included the deprecation notices for adminpack. 
@theadamwright  can you confirm?

Reply:
That's right, DRITA is a part of EPAS and we never added pgAgent support for PGE.